nengo/processes.py

```diff
@@ -66,17 +65,16 @@ class FilteredNoise(Process):
 
     Parameters
     ----------
-    synapse : Synapse, optional
-        The synapse to use to filter the noise. Default: Lowpass(tau=0.005)
-    dist : Distribution, optional
+    synapse : Synapse, optional (Default: ``Lowpass(tau=0.005)``)
+        The synapse to use to filter the noise.
+    dist : Distribution, optional (Default: ``Gaussian(mean=0, std=1)``)
         The distribution used to generate the white noise.
-        Default: Gaussian(mean=0, std=1)
-    scale : bool, optional
+    scale : bool, optional (Default: True)
         Whether to scale the white noise for integration, making the output
-        signal invariant to `dt`. Defaults to True.
-    synapse_kwargs : dict, optional
-        Arguments to pass to `synapse.make_step`.
-    seed : int, optional
+        signal invariant to ``dt``.
+    synapse_kwargs : dict, optional (Default: None)
```

Comment:
Not sure, this is what I've used more often in other docstrings... usually I say something like `If None, ...` but in this case it's pretty obvious that no arguments will be passed if you pass `None`, I think.